core/controllers/suggestion.py

```diff
@@ -83,13 +89,65 @@ class SuggestionHandler(base.BaseHandler):
 
     @acl_decorators.can_suggest_changes
     def post(self):
+        change_dict = self.payload.get('change')
+        target_id = self.payload.get('target_id')
+        target_type = self.payload.get('target_type')
+        target_version_at_submission = self.payload.get(
+            'target_version_at_submission')
+        suggestion_type = self.payload.get('suggestion_type')
+        description = self.payload.get('description')
+
+        suggestion_domain_class = (
+            suggestion_registry.SUGGESTION_TYPES_TO_DOMAIN_CLASSES[
+                suggestion_type])
+        # While creating the below domain object fields suggestion_id and
+        # score_category is kept None because values for these fields are
```

Comment:
I don't really understand this. Why don't we just set them as the right things? (I don't understand what the reason given has to do with the conclusion that we need to set the fields equal to None.)

Reply:
Actually what we were trying to save the images for suggestions before the suggestion is actually created and the reason why these fields were None is that these are created only while the actual creation of the suggestion. 
But after a discussion with @DubeySandeep, it was found that making these not None will require considerable refactoring.
Also, this PR is required to be merged ASAP because it is blocking#10469. For now, I've added a todo and reverted it back to save the images after the suggestions are created.